vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/GridMultiSelectionModel.java

```diff
@@ -58,7 +62,7 @@
         HIDDEN,
 
         /**
-         * By default select all checkbox depends on the grid's dataprovider.
+         * By default, select all checkbox depends on the grid's data provider.
```

Comment:
What is a data provider ? The user should not need to know anymore. Instead
"By default, the visibility of the select all checkbox depends on how the grid's items are fetched:"

Reply:
Done